features/external_users/advocate_new_claim.feature

```diff
@@ -1,275 +0,0 @@
-      And I submit to LAA
-     Then I should be redirected to the claim certification page
-      And I fill in the certification details and submit
-     Then I should be redirected to the claim confirmation page
-      And I should see the claim totals accounting for the miscellaneous fee
-
-  Scenario: Fixed Fee case type does not save Initial Fees
-     Given I am a signed in advocate
-       And There are case types in place
-       And I am on the new claim page
-      When I fill in the claim details
-       And I fill in a Miscellaneous Fee
-       And I select a Case Type of "Fixed fee"
-       And I submit to LAA
-      Then There should not be any Initial Fees saved
-       And There should be a Miscellaneous Fee Saved
-
-  Scenario: Non-Fixed Fee case type does not save Fixed Fees
-     Given I am a signed in advocate
-       And There are case types in place
-       And I am on the new claim page
-      When I fill in the claim details
-       And I fill in a Fixed Fee
-       And I select a Case Type of "Trial"
-       And I submit to LAA
-      Then There should not be any Fixed Fees saved
-
-  Scenario: Edit existing non-Fixed case type to be Fixed
-    Given I am a signed in advocate
-      And There are case types in place
-      And a non-fixed-fee claim exists with basic and miscellaneous fees
-     When I am on the claim edit page
-      And I select a Case Type of "Fixed fee"
-      And I submit to LAA
-     Then There should not be any Initial Fees saved
-      And There should be a Miscellaneous Fee Saved
-
-  @javascript @webmock_allow_localhost_connect
-  Scenario Outline: Daily attendance fees derived from actual length of Trial
-  Given I am a signed in advocate
-    And There are case types in place
-    And I am on the new claim page with Daily Attendance Fees in place
-    And I select2 a Case Type of "Trial"
-    And I fill in actual trial length with <actual_trial_length>
-   Then The daily attendance fields should have quantities <daf_quantity>, <dah_quantity>, <daj_quantity>
-
-  Examples: Actual Trial lengths and how the Daily attendance fields should break it down
-    | actual_trial_length   | daf_quantity | dah_quantity | daj_quantity |
-    | 1                     | 0            | 0            | 0            |
-    | 2                     | 0            | 0            | 0            |
-    | 3                     | 1            | 0            | 0            |
-    | 40                    | 38           | 0            | 0            |
-    | 41                    | 38           | 1            | 0            |
-    | 50                    | 38           | 10           | 0            |
-    | 51                    | 38           | 10           | 1            |
-    | 60                    | 38           | 10           | 10           |
-    | 70                    | 38           | 10           | 20           |
-
-  #TODO could potentially merge this Scenario:with the one above
```

Comment:
Let's make a note in a TODO file that this needs to be tested in some sort of javascript testing